src/model/FaceliftEnum.cpp

```diff
@@ -32,10 +32,14 @@
 #include <QDebug>
 
 namespace facelift {
+    
+namespace Enum {
```

Comment:
In facelift, namespaces are used to group entities which belong together into a layer. A upper-level layer (such as "facelift") should not have any dependency to a lower layer (such as "facelift::ipc"). Your change is breaking that rule.
Also, if you introduce a "Enum" namespace (whose name is also not right), you should also introduce a "Struct" namespace, but I do not see any benefit in doing so anyway.

Reply:
Sorry, but coding standards/guidelines in facelift, also community page, etc. are EMPTY. This means that you cannot expect any coding standards from anyone. 

Also separate namespace for utility makes sense and is a common practice in the software world, especially when you do not want to create a class/struct with all static methods.

Reply:
>In facelift, namespaces are used to group entities which belong together into a layer. A upper-level layer (such as "facelift") should not have any dependency to a lower layer (such as "facelift::ipc"). Your change is breaking that rule.

in this case namespace is used for its straight purpose - name scope.
Any other options like different name, class or structure are worst alternatives


>Also, if you introduce a "Enum" namespace (whose name is also not right), 

please suggest a better name

>you should also introduce a "Struct" namespace, but I do not see any benefit in doing so anyway.

it's not needed at the moment

Reply:
actually StringConversionHandler is not needed at all
and in the future it should be replaced with toString<class T> template function
with different specifications and SFINAE principle.
Also @bitmouse suggested the idea to move the enums stuff into a separate folder.